requirements.txt

```diff
@@ -1,4 +1,4 @@
 ops
 pydantic
 # crossplane is a package from nginxinc to interact with the Nginx config
```

Comment:
more useful to add a comment about what module needs it

Reply:
I actually haven't added any new dependency, not sure why it's showing that there's a change in the file

Reply:
True, this is a dependency I added, and it's needed to build the Nginx config